packages/admin-next/dashboard/src/components/table/table-cells/order/fulfillment-status-cell/fulfillment-status-cell.tsx

```diff
@@ -1,4 +1,4 @@
-import type { FulfillmentStatus } from "@medusajs/medusa"
+import { FulfillmentStatus } from "@medusajs/medusa"
```

Comment:
The admin should not import `@medusa/medusa`, should it?

Reply:
Yes, good catch this was wrong autoimport